charts/linkerd2-cni/templates/cni-plugin.yaml

```diff
@@ -148,12 +148,16 @@ data:
         "outgoing-proxy-port": {{.Values.outboundProxyPort}},
         "proxy-uid": {{.Values.proxyUID}},
         "ports-to-redirect": [{{.Values.portsToRedirect}}],
+        {{- if .Values.ignoreInboundPorts }}
         "inbound-ports-to-ignore": [
           {{- include "partials.splitStringList" .Values.ignoreInboundPorts -}}
         ],
+        {{- end }}
+        {{- if .Values.ignoreOutboundPorts }}
         "outbound-ports-to-ignore": [
           {{- include "partials.splitStringList" .Values.ignoreOutboundPorts -}}
         ],
+        {{- end }}
```

Comment:
If `ignoreInboundPorts` and `ignoreOutboundPorts` are no longer used, why keeping this section?

Reply:
@alpeb My thought was with CNI installation if a user wants to specify these than we'd still need to populate those configurations—even thought that's not the default anymore.

Reply:
Ok, so that'd mean keeping `_proxy-init.tpl` as-is as well, otherwise why keeping those options just for CNI?

Reply:
Yep so I'll keep it here as well as `_proxy-init.tpl` 👍